README.md

```diff
@@ -676,12 +674,14 @@ You can install an import map for your application using a `<script>` element, e
 ```
 
 ```html
-<script type="importmap" src="import-map.json"></script>
+<script type="importmap" src="import-map.importmap"></script>
```

Comment:
If this file extension is tentative, it could be worth including an additional note that the final file extension is still being decided. I hope it would be registered with IANA as well, as web servers would definitely need to support this. And I hope we can get that process going sooner rather than later too.

Reply:
File extension isn't intended to be specified; people can use whatever. Maybe I'll just omit it to be clearer.

Reply:
Hmm, that would be somewhat of an issue for adoption.

Reply:
If there’s a mime type, there should be an extension, whether browsers care about it or not - otherwise there’s no explicit reliable signal for servers to know which mime type to serve.

Reply:
I'm happy to leave that up to folks who work on servers to figure out.